include/hse/hse.h

```diff
@@ -679,7 +679,7 @@ hse_kvs_prefix_delete(
  * @param kvs: KVS handle from hse_kvdb_kvs_open().
  * @param flags: Flags for operation specialization.
  * @param txn: Transaction context (optional).
- * @param key: Key to put into @p kvs.
+ * @param key: Key to put into the KVS.
```

Comment:
I never really liked the current wording.  It's too wordy and unnecessarily formal.   There's no need to be so formal when there are no reasonable misinterpretations.  For example, we don't need to write
`@param kvs: KVS handle from hse_kvdb_kvs_open()` because there's no other way to get a valid KVS handle except via hse_kvdb_kvs_open().  There'd be no loss of information if we wrote "a KVS handle", or "a handle to an open KVS". 

Along the same lines, the `@brief` can be simplified to:   `@brief Put a key-value pair into a KVS`.  Nobody will wonder which KVS since a KVS handle is the first function parameter.

Also, key and value descriptons could simply be  `@param key: the key` and `@param value: the value`.  In the context of the full comment it is clear that the key and value are going into the KVS and that the value is associated with the key  (it's a pair!).

And the remarks..  Some of them are a waste of space.  "kvs must not be NULL".  Okay thanks, that was helpful :)

Reply:
I like your suggestions. Will take them into account. Please let me know what remarks you think are verbose. Tried to keep the remarks to issues with parameters for the most part.